gradle.properties

```diff
@@ -3,7 +3,7 @@ group=io.ballerina.stdlib
 version=2.3.0-SNAPSHOT
 puppycrawlCheckstyleVersion=8.18
 bouncycastleVersion=1.69
-ballerinaGradlePluginVersion=0.14.1
+ballerinaGradlePluginVersion=1.0.0
```

Comment:
As I remember, dont we have to enable graalvm flag in the plugin properties?

Reply:
The flag is enabled in the workflow when calling `grade build` : https://github.com/ballerina-platform/ballerina-standard-library/blob/6d725e5ca168ca4826b26fb7a38cf975ad0c2559/.github/workflows/build-with-bal-test-native-template.yml#L91

Reply:
Ack